recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
```

Comment:
Why don't we use the official releases as listed in https://pcsclite.apdu.fr/#download?

Reply:
I can do this as well, and I think again as per @BastianZim I used the GH instead but yeah I can point it here instead

Reply:
Ah let's wait for @BastianZim's reply before any change, then. It's unclear to me why we need this. The CF bot can detect new version updates even if the source is not from GH.

Reply:
I'll go ahead and change it for now, easy enough to revert if needed